tools/hive/ubuntu/Dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM ubuntu:trusty
```

Comment:
This is pretty old version of ubuntu. Furthermore, all this image seems to do is add the SSD Daemon (which I'm not sure if we need), and set environment variables that are later overwritten.